src/Shyim/PluginInstaller.php

```diff
@@ -207,14 +229,21 @@ private static function apiRequest($path, $method, array $params = [])
      * @param string $url
      * @param string $name
      * @param string $version
+     *
+     * @throws \Exception
      */
     private static function downloadAndMovePlugin($url, $name, $version)
     {
-        $ch = curl_init($url);
-        curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
-        curl_setopt($ch, CURLOPT_FOLLOWLOCATION, 1);
-        $content = curl_exec($ch);
-        curl_close($ch);
+        $content = self::makePluginHTTPRequest($url);
+
+        if (substr($content, 0, 1) == "{" && substr($content, strlen($content) - 1, 1) == "}") {
```

Comment:
Could we not check here the response Content-Type?

Reply:
sure of course. ive just updated the pr.